src/ducks/apps/components/ApplicationPage/HeaderShortcutActions.jsx

```diff
@@ -0,0 +1,98 @@
+import cozySmileIcon from 'assets/icons/icon-cozy-smile.svg'
+import { buildFileFromPathQuery } from 'ducks/queries'
+import React, { useState } from 'react'
+
+import { useClient, useFetchShortcut } from 'cozy-client'
+import { move } from 'cozy-client/dist/models/file'
+import flag from 'cozy-flags'
+import Button from 'cozy-ui/transpiled/react/Buttons'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import { useAlert } from 'cozy-ui/transpiled/react/providers/Alert'
+import useBreakpoints from 'cozy-ui/transpiled/react/providers/Breakpoints'
+import { useI18n } from 'cozy-ui/transpiled/react/providers/I18n'
+
+const HeaderShortcutActions = ({ app }) => {
+  const { isMobile } = useBreakpoints()
+  const { t } = useI18n()
+  const client = useClient()
+  const { shortcutInfos } = useFetchShortcut(client, app.id)
+  const { showAlert } = useAlert()
+
+  const [isBusy, setBusy] = useState(false)
+
+  const config = flag('store.alternative-source')
+
+  const handleAdd = async () => {
+    try {
+      setBusy(true)
+      // Install can be disabled
+      const shortcutCategory = app.categories[0]
+      const categoryPath = config.categories[shortcutCategory]
+      const fileQuery = buildFileFromPathQuery(categoryPath)
+      const categoryDirectory = await client.fetchQueryAndGetFromState(
+        fileQuery
+      )
+      await move(client, app, categoryDirectory.data[0])
+      showAlert({
+        message: t('HeaderShortcutActions.addSuccess'),
+        severity: 'secondary'
+      })
+    } finally {
+      setBusy(false)
+    }
```

Comment:
Is that expected that we don't catch anything? Don't we want to show an error message to the user?

Reply:
You're right, I added an alert to warn the user